packages/stash/src/react/useRecords.ts

```diff
@@ -0,0 +1,22 @@
+import { Table } from "@latticexyz/config";
+import { Key, Stash, TableRecord } from "../common";
+import { useStash } from "./useStash";
+import { getRecords } from "../actions/getRecords";
+import { isArrayEqual } from "./isArrayEqual";
+
+export type UseRecordsOptions<table extends Table = Table> = {
+  stash: Stash;
+  table: table;
+  keys?: readonly Key<table>[];
+};
+
+export type UseRecordsResult<table extends Table = Table> = readonly TableRecord<table>[];
```

Comment:
In almost all cases, I want to work with an array of records, not an object.

Doing this inside a `useStash` selector means we can memoize the results and avoid things like `Object.values` everywhere.